src/main/java/de/hpi/bpt/chimera/execution/controlnodes/activity/EmailActivityInstance.java

```diff
@@ -109,15 +85,15 @@ private void sendMail() {
 		log.info("sending a Mail");
 		// TODO
 		Email email = new SimpleEmail();
-		email.setHostName(hostName);
-		email.setSmtpPort(port);
-		email.setAuthenticator(new DefaultAuthenticator(sendMail, password));
+		email.setHostName(this.hostName);
+		email.setSmtpPort(this.port);
+		email.setAuthenticator(new DefaultAuthenticator(this.sendMail, this.password));
 		email.setSSLOnConnect(true);
 		try {
-			email.setFrom(sendMail);
-			email.setSubject(subject);
-			email.setMsg(message);
-			email.addTo(receiverMail);
+			email.setFrom(this.sendMail);
```

Comment:
I am just wondering why you added all of this unnecessary 'this'. I did not count, but in total there are ~30 in this PR.

Reply:
Added for readability. For me as I was unfamiliar with the code, it was not obvious that these variables are member variables.